### DISCUSSION TITLE
What is '-' in REFERRER URLS chart

### DISCUSSION THREAD

**Discussion by loan-mgt** [allinurl/goaccess#2714] [Q&A]
I've set up GoAccess to monitor some information about what is going on in my home lab.

However, there's a part of the REFERRER URLS chart that I don't understand.  92% of the request hits "-", instead of a domain name. What does this "-" mean?

Does it indicate a direct call from an IP address?

Thanks in advance for the insight.

![image](https://github.com/user-attachments/assets/d71b34de-0e8f-4267-b537-df82955d35ee)

**Comment by allinurl** [comment]
Assuming you have the proper log format, it looks like many of the hits or requests in your log are missing a referer in the header, indicating direct traffic. There could be various reasons for this. I’d recommend checking the IP address with the highest traffic to see if it might be associated with bots.
